news/disable-system-clipboard.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* Disabled PTK copying/pasting from system clipboard in VI mode by default (can be re-enabled with ``$XONSH_USE_SYSTEM_CLIPBOARD``).
+
+**Deprecated:**
+
+* Deprecated ``$XONSH_COPY_ON_DELETE`` in favor of ``$XONSH_USE_SYSTEM_CLIPBOARD``.
```

Comment:
There should be a reason to deprecate the current behaviour. Also a warning msg need to be added for users.